libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -703,17 +699,19 @@ - (void)beginTokenEndpointFlow:(SFOAuthTokenEndpointFlow)flowType {
     NSData *encodedBody = [params dataUsingEncoding:NSUTF8StringEncoding];
     [request setHTTPBody:encodedBody];
     
-    // We set the timeout value for NSMutableURLRequest above, but NSMutableURLRequest has its own ideas
-    // about managing the timeout value (see https://devforums.apple.com/thread/25282).  So we manage
-    // the timeout with an NSTimer, which gets started here.
-    [self startRefreshFlowConnectionTimer];
+    request.timeoutInterval = self.timeout;
```

Comment:
Do we even need this now, given that it's set in [Line 650](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1969/files#diff-a19763c4fa428fe97ccc907d35e442c3R650)?